src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -1612,6 +1629,17 @@ SubCmdValidate::execute(const SubCmdOptions& _options) const
     for (auto &userTestName : testsToRun)
       boost::algorithm::to_lower(userTestName);   // Lower case the string entry
 
+    // check if xclbin folder path is provided
+    if (!xclbin_location.empty()) {
+      XBU::verbose("Sub command: --path");
+      if (!boost::filesystem::exists(xclbin_location) || !boost::filesystem::is_directory(xclbin_location))
+        throw xrt_core::error((boost::format("Invalid directory path : '%s'") % xclbin_location).str());
+      if(xclbin_location.compare(".") == 0 || xclbin_location.compare("./") == 0)
```

Comment:
Curious all these `if(` in the neighborhood?
It would be nice to have a correctly configured IDE/editor/clang-format/...